### ISSUE TITLE
Tower of Ascension Quest is Broken 

### ISSUE THREAD

**Issue by MCminion** [Wynncraft/Issues#1841]
Hey Wynncraft,

I have started the ToA quest.  On this one day, I couldn't go on to the next floor on the floors (yea sounds confusing) what I mean is that i can't move on thru the quest.  Please let the staff know that this quest is broken, Thanks!

MCminion0930

**Comment by Trixter313** [comment]
Duplicate of https://github.com/Wynncraft/Issues/issues/1774

**Comment by epicman739** [comment]
Try relogging, restarting, etc.
Some people say that repeatedly jumping at the teleport spot helps.

**Comment by TxMelonMan** [comment]
MCminion...this happens for me all the time...i hard a solution called "waiting" after waiting on the teleported for about 10 min i teleported but had to do the same thing again...

**Comment by MCminion** [comment]
Tama why did u close the issue?
